File: app/src/main/java/com/fstyle/structure_android/utils/common/StringUtils.java
package com.fstyle.structure_android.utils.common;

/**
 * Created by framgia on 27/04/2017.
 */

public final class StringUtils {

    private StringUtils() {
        // No-op
    }

    public static boolean isBlank(String input) {
        return input == null || input.isEmpty();
    }

    public static boolean isNotBlank(String input) {
        return !isBlank(input);
    }

    public static int convertStringToNumber(String input) {
        try {
            return Integer.parseInt(input);
        } catch (NumberFormatException e) {
            return Integer.MIN_VALUE;
        }
    }
}

File: app/src/main/java/com/fstyle/structure_android/utils/common/StringUtils.kt
package com.fstyle.structure_android.utils.common

/**
 * Created by framgia on 05/05/2017.
 */

object StringUtils {

  fun isBlank(input: String): Boolean {
    return input.isEmpty()
  }

  fun isNotBlank(input: String): Boolean {
    return !isBlank(input)
  }

  fun convertStringToNumber(input: String?): Int {
    try {
      return Integer.parseInt(input)
    } catch (e: NumberFormatException) {
      return Integer.MIN_VALUE
    }
  }
}
